keras/src/quantizers/gptqutils.py

```diff
@@ -0,0 +1,386 @@
+import io
+import logging
+import random
+import tarfile
+import time
+
+import numpy as np
+import requests
+from datasets import load_dataset
```

Comment:
Maybe we should discuss the decision for adding a dependency on the `datasets` package. Not sure if this should be required, as it is not essential to the overall feature.

Reply:
Have tried to keep the interfaces same as hugging face so need this, but again this also can be up for discussion @divyashreepathihalli @FabianToSpace  please comment

Reply:
Sure makes perfect sense have removed the hugging face dependency and updated the colab as well